.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        jdk: [ 8, 11 ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{matrix.jdk}}
+      - name: Set up Python 3
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+      - name: Install Python dependencies
+        run: |
+          python -m pip install --upgrade pip
+          if [ -f .travis-requirements.txt ]; then pip install -r .travis-requirements.txt; fi
+      - name: Security Demo Build & Setup
+        run: |
+          owner=${{ github.repository_owner }}
+          echo "REPO_SITE=${owner}" >> $GITHUB_ENV
```

Comment:
I am not quite sure whether this will work --- AFAIK, `$GITHUB_ENV` may only work outside bash script. Could you please confirm this by setting `REPO_SITE` to a value except `opprop`, and check whether this value is passed correctly to `setup.sh`.

Or you can try setting environment variables in this way:

```
jobs:
  weekday_job:
    runs-on: ubuntu-latest
    env:
      DAY_OF_WEEK: Mon
    steps:
      - name: "Hello world when it's Monday"
        if: env.DAY_OF_WEEK == 'Mon'
        run: echo "Hello $FIRST_NAME $middle_name $Last_Name, today is Monday!"
        env:
          FIRST_NAME: Mona
          middle_name: The
          Last_Name: Octocat
```

Reply:
Thanks for pointing it out! Yes, as you said, the new environment variable inserted to `GITHUB_ENV` is not activated until the next step begins.